recipe/migrations/glib2646.yaml

```diff
@@ -0,0 +1,11 @@
+migrator_ts: 1
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+
+glib:
```

Comment:
What is the right pin here? Do we need a pin both libglib and glib?

Reply:
I  would expect that for building you always need glob installed. We can pin both to be sure but we shouldn't need to.